tests/regression/test_interpolate.py

```diff
@@ -174,3 +175,89 @@ def test_lvalue_rvalue():
     u.assign(1.0)
     u.interpolate(u + 1.0)
     assert np.allclose(u.dat.data_ro, 2.0)
+
+
+@pytest.mark.parametrize("degree", range(1, 6))
+def test_interpolator_spectral(degree):
+    mesh = UnitSquareMesh(10, 10, quadrilateral=True)
+    x = SpatialCoordinate(mesh)
+    fe1 = FiniteElement("CG", mesh.ufl_cell(), degree, variant="spectral")
```

Comment:
`"spectral"` is now the default
Do we want to test equispaced and spectral?

Reply:
Equispaced now included on quads and triangles.